src/routes/v2/project_creation.rs

```diff
@@ -889,6 +894,27 @@ async fn project_create_inner(
     }
 }
 
+async fn insert_project_create_event(
```

Comment:
i think this should be inlined since it's not used anywhere else

Reply:
I disagree because I think it improves the readability. If you don't care about the event insertion while you're reading through `project_create_inner`, you can see the name and move on, giving it no further thought. If you do care, it has a descriptive name telling you what it does and is a simple F12/`View Declaration` away and then you get to focus just on the details of how we generate the event, without the distractions of the all the other things we're doing in that method.

I do agree that the benefit here is relatively slim because it ended up being a pretty short method, but I still think it's easier to read. If you'd really like, though, I'll inline it.

Reply:
Yeah that seems fine